app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx

```diff
@@ -20,6 +20,17 @@ const rainbow = [
 	'#760089'
 ];
 
+const colours = {
```

Comment:
There's a lint failure on this (you can see it if you run `yarn lint`) due to the spacing. That's probably okay though if you go with the colour picker because this object can just be deleted.